components/BiospecimenTable.tsx

```diff
@@ -33,12 +28,18 @@ export const BiospecimenTable: React.FunctionComponent<IBiospecimenTableProps> =
             wrap: true,
             sortable: true,
         },
-        {
-            name: 'Biospecimen Type',
-            selector: 'BiospecimenType',
-            wrap: true,
-            sortable: true,
-        },
+        ...generateColumnsByDataSchema(
+            'bts:Biospecimen',
+            props.schemaDataMap,
+            // need to add a custom sort function for the id
+            {
+                HTANBiospecimenID: {
+                    sortFunction: sortByBiospecimenId,
+                },
+            },
+            // Component seems to be always "Biospecimen", no need to have a column for it
+            ['Component']
```

Comment:
looks like the columns will be reordered, so now HTANBiospecimenId is no longer the first column, right? its okay?

Reply:
Right, if the column order is important we will need to do some manual sorting.